modules/traefik/external.tf

```diff
@@ -172,7 +172,7 @@ resource "aws_autoscaling_attachment" "external" {
 # Everything else deployed should alias (recommended) or CNAME this domain
 resource "aws_route53_record" "external_dns_record" {
   zone_id = data.aws_route53_zone.default.zone_id
-  name    = var.traefik_external_base_domain
+  name    = var.traefik_external_route53_domain
```

Comment:
The variable name change here doesn't have the corresponding change in [variables.tf](https://github.com/dsaidgovsg/terraform-modules/blob/chore--es-remove-unused-var/modules/traefik/variables.tf#L13-L23)?

The CI also caught similar stuff:

```
╷
│ Error: Reference to undeclared input variable
│ 
│   on external.tf line 175, in resource "aws_route53_record" "external_dns_record":
│  175:   name    = var.traefik_external_route53_domain
│ 
│ An input variable with the name "traefik_external_route53_domain" has not been declared. This variable can be declared with a variable "traefik_external_route53_domain" {} block.
╵
╷
│ Error: Reference to undeclared input variable
│ 
│   on internal.tf line 176, in resource "aws_route53_record" "internal_dns_record":
│  176:   name    = var.traefik_internal_route53_domain
│ 
│ An input variable with the name "traefik_internal_route53_domain" has not been declared. This variable can be declared with a variable "traefik_internal_route53_domain" {} block.
╵
```

Reply:
After checking, this change is not needed. Removed the change in vars on `traefik`